x-pack/plugins/siem/public/management/pages/policy/store/policy_list/reducer.ts

```diff
@@ -4,26 +4,24 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { PolicyListState } from '../../types';
 import { isOnPolicyListPage } from './selectors';
 import { ImmutableReducer } from '../../../../../common/store';
 import { AppAction } from '../../../../../common/store/actions';
 import { Immutable } from '../../../../../../common/endpoint/types';
+import { PolicyListState } from '../../types';
 
-export const initialPolicyListState = (): PolicyListState => {
-  return {
-    policyItems: [],
-    isLoading: false,
-    apiError: undefined,
-    pageIndex: 0,
-    pageSize: 10,
-    total: 0,
-    location: undefined,
-  };
+export const initialPolicyListState: Immutable<PolicyListState> = {
+  policyItems: [],
+  isLoading: false,
+  apiError: undefined,
+  pageIndex: 0,
+  pageSize: 10,
+  total: 0,
+  location: undefined,
 };
 
 export const policyListReducer: ImmutableReducer<PolicyListState, AppAction> = (
-  state = initialPolicyListState(),
+  state = initialPolicyListState,
```

Comment:
oh man. this makes me nervous (for runtime) - 😟 Sure hope TS checker never fails us and somehow this is mutated at runtime 😆

Reply:
I should have expressed myself better, but that's what I meant here: https://github.com/elastic/kibana/pull/67768/files/4cc07654ce545570dc29528ce5f75bee9bd3b166#r433907302

Trying to think through this:

* From the reader's perspective, the value is `Immutable`, and so returning it from a function is pointless.
* However, if something goes wrong w/ `Immutable`, then this could cause unexpected results
* 'policy_details' [intentionally mutates state](https://github.com/oatkiller/kibana/blob/cherry-time/x-pack/plugins/siem/public/management/pages/policy/store/policy_details/reducer.ts#L102)

With that in mind, maybe I should revert this change. And maybe we should reconsider using `Immutable` in 'policy_details', since we know we are mutating state.

Reply:
policy list and detail initial state is returned from a function now.

Reply:
👍